spec/static/spec/v0.9/api.html

```diff
@@ -4,20 +4,62 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <link rel="icon" href="/api/stac/static/favicon.ico">
-    <title>STAC API</title>
+    <link rel="stylesheet" href="/api/stac/static/style/css/admin.css" type="text/css">
+    <link rel="icon" href="/api/stac/static/style/img/ico/favicon.ico">
+    <title>data.geo.admin.ch STAC API</title>
+    <style>
+        html, body {
+            overflow-x:visible;
+        }
+        #content, .row {
+            margin-right: 0px;
+            margin-left: 0px;
+        }
+    </style>
 </head>
 
 <body>
-    <div id="content"></div>
-    <script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"> </script>
-    <script type="text/javascript">
-        Redoc.init('openapi.yaml', {
-            expandResponses: "200,201,202,203,204",
-            pathInMiddlePanel: true,
-            hideDownloadButton: true
-        }, document.getElementById('content'));
-    </script>
+
+        <!-- <div class="container container-main nav-open"> -->
+            <header style="border-bottom:4px solid #dc0018;">
+                <a href="#" class="brand hidden-xs" title="Back to home">
+                    <img src="/api/stac/static/style/img/logo-CH.svg" onerror="this.onerror=null; this.src='/api/stac/static/style/img/logo-CH.png'" alt="Back to home" />
+                    <h1>data.geo.admin.ch STAC API</h1>
+                </a>
+            </header>
```

Comment:
Wrong indentation

Reply:
Apart from the indentation, I corrected everything we spoke of this morning. It's now deployed on dev.